NetflixClone/Classes/Networking/NetworkConfiguration.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  NetworkConfiguration.swift
+//  NetflixClone
+//
+//  Created by Joseph Ching on 2022-09-19.
+//
+
+import Foundation
+
+
+class NetworkConfiguration {
```

Comment:
Let's make it an enum, Since we don't need to allow initialization of the configurations object.